.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,76 @@
+name: Bug Report
+description: Create a report with a procedure for reproducing the bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Check [CONTRIBUTING guideline](https://github.com/fluent/fluentd/blob/master/CONTRIBUTING.md) first and here is the list to help us investigate the problem.
+  - type: textarea
```

Comment:
Replace the link please.

Reply:
But I didn't found a `CONTRIBUTING` file in this repository could to replace.

Reply:
But I didn't found a `CONTRIBUTING` file in this repository could to replace.

Reply:
Let's copy one CONTRIBUTING file to this repo :)

Reply:
Whether the link (https://github.com/fluent/fluent-operator#contributing) can be replaced in this place?

Reply:
Whether the link (https://github.com/fluent/fluent-operator#contributing) can be replaced in this place?

Reply:
Ok.

Reply:
> Whether the link (https://github.com/fluent/fluent-operator#contributing) can be replaced in this place?

Yes

Reply:
Done. add a `CONTRIBUTING` file to replace.